Swift/Select.swift

```diff
@@ -27,9 +27,17 @@ public final class Select: Query, FromRouter {
     /// - Parameter dataSource: The DataSource object.
     /// - Returns: The From object that represent the FROM clause of the query.
     public func from(_ dataSource: DataSourceProtocol) -> From {
+        
+        var db: Database!
+        if let colSource = dataSource.source() as? Collection {
+            db = colSource._db
```

Comment:
Maybe we should have a proper internal property for the _db.

Reply:
I created a getter from Swift.